tests/integration/sandbox.py

```diff
@@ -641,7 +650,7 @@ def raft_retry(self, func):
                     if no_leader_first:
                         LOG.info("-CLUSTERDOWN response received, will retry"
                                  " for %s seconds", self.noleader_timeout)
-                        #no_leader_first = False
+                        # no_leader_first = False
```

Comment:
do we still need this comment?

Reply:
Not sure. If we delete the comment, we can also delete the if check. If we do that, we probably need to fix error message as well, it is not accurate. As this is becoming something unrelated, it can be part of another PR.